themes/luci-theme-rosy/htdocs/luci-static/rosy/js/script.js

```diff
@@ -272,15 +272,21 @@
         }
         var a = window.getComputedStyle($(".cbi-section .table .tr:first-child")[0], ':before').getPropertyValue('content');
         var b = window.getComputedStyle($('.cbi-section .table .tr:last-child')[0], ':before').getPropertyValue('content');
-        console.log(typeof a);
         
         if( a != 'none' || b != 'none' ){
             $('.cbi-section .table .tr:first-child .td:first-child').css('border-top-left-radius', '0');
             $('.cbi-section .table .tr:last-child .td:first-child').css('border-bottom-left-radius', '0');
             $('.cbi-section .table .tr:first-child .th:first-child').css('border-top-left-radius', '0');
-            console.log($('.cbi-section .table .tr:last-child .td:first-child'));
             
         }
     });
 
+    $('<samll>').text($('.cbi-progressbar').attr('title')).appendTo('.cbi-progressbar');
+    setInterval(function(){
+        $('.cbi-progressbar').each(function(i, e){
+            $(this).children("samll").text($(this).attr('title'));
+        });
+        
+    }, 5000);
```

Comment:
What is the purpose of this code? It appears to do nothing at all. The selector is misspelled, the `.attr()` call with only one argument is a getter and not a setter and the resulting rvalue is not used at all.

Why are you reading the `title` attributes of all progressbar elements every 5s?

Reply:
![image](https://user-images.githubusercontent.com/39110824/49208815-bd19fe80-f3f3-11e8-9ff4-2cdf04385a83.png)
Because the page just loaded, I can't read the content of the progress bar immediately, so I can only let him delay reading, and read it once every five seconds. This can display the data after each refresh of the page.
Sorry, my English is not very good. If you bring dyslexia, please forgive me.

Reply:
Ok, I have fixed the relevant code, please check